ops/docker/Dockerfile.indexer

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15-alpine3.13 as builder
```

Comment:
We should use the same docker image that the batch submitter is using so that caching can work better
https://github.com/ethereum-optimism/optimism/blob/e0f7e5f43388d489de8882c316c24b17d489a24d/ops/docker/Dockerfile.batch-submitter-service#L1

Reply:
Oh sure, will update 👍